addOns/scripts/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
 ### Changed
+- Focus only on Text-Based scripts to avoid recording issues with client scripts.
```

Comment:
Should be tweaked to mention instead that it's possible to display a script without focusing.

Reply:
PR title should also be updated.